### PR TITLE
Bank Commit

### PR DESCRIPTION
**Pull request by dinhjeffrey**

### CODE REVIEW

File: lib/Account.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 module.exports = Account;
 
-function Account(name, balance) {
+function Account(name, balance) 
+{
   // We take the name that is passed in or we default to 'John Doe'
-  this.name = name = || 'John Doe';
+  this.name = name = || 'John Doe';
```
Comment (paulot) [inline lib/Account.js]: this.name = name || 'John Doe'

no equal's

File: lib/Account.js [hunk 2]
```diff
@@ -11,17 +12,21 @@ function Account(name, balance) {
...
  *
- * @param amount: The amount to be withdrawn from the bank acocunt.
+ * @param amount: The amount to be withdrawn from the bank account.
  */
-Account.prototype.withdraw = function(amount) {
-  // Complete funciton here
+Account.prototype.withdraw = function(amount) 
+{
+  // Complete function here
+  this.balance -= amount;
```
Comment (paulot) [inline lib/Account.js]: Prevent the balance from going negative. So if the balance will be negative, don't do anything.

Comment (dinhjeffrey) [inline lib/Account.js]: okay I've revised it in the new push

Comment (dinhjeffrey) [inline lib/Account.js]: i think my if condition may not be written correctly

**Pull request merged**